feat: drop WITH(KEY) syntax (#5363)
confluentinc/ksql: docs/developer-guide/create-a-stream.md

For runtime statistics and query details run: DESCRIBE EXTENDED <Stream,Table>;
```

You may notice that ksqlDB has added a key column named `ROWKEY`.

> This is the default key column added if you do not provide one.

Reviewer: ```suggestion
This is the default key column that ksqlDB adds if you don't provide one.
```

> If your data does not contain a Kafka serialized

Reviewer: ```suggestion
If your data doesn't contain a {{ site.ak }} serialized
```

> `STRING` in the Kafka message key you should not use `ROWKEY` in your SQL statements,

Reviewer: ```suggestion
`STRING` in the {{ site.ak }} message key, don't use `ROWKEY` in your SQL statements,
```

> as the behaviour will be undefined.

Reviewer: ```suggestion
because this may cause unexpected results.
```

### Create a Stream with a Specified Key

> The previous SQL statement does not define a column to represent the data in the

Reviewer: ```suggestion
The previous SQL statement doesn't define a column to represent the data in the
```

> Kafka message key in the underlying Kafka topic. The system therefore added a

Reviewer: ```suggestion
{{ site.ak}} message key in the underlying {{ site.ak}} topic, so the system adds a
```

`ROWKEY` column with type `STRING`.

> Where the Kafka message key is serialized in a key format ksqlDB supports, (currently only `KAFKA`),

Reviewer: ```suggestion
If the {{ site.ak }} message key is serialized in a key format that ksqlDB supports (currently `KAFKA`),
```

you can specify the key in the column list of the CREATE STREAM statement.

> For example, the Kafka message key of the pageviews topic is a `BIGINT` containing the `viewtime`.

Reviewer: ```suggestion
For example, the {{ site.ak }}  message key of the `pageviews` topic is a `BIGINT` containing the `viewtime`.
```

> So, you can write the CREATE STREAM statement like this:

Reviewer: ```suggestion
so you can write the CREATE STREAM statement like this:
```
